apps/web-giddh/src/app/vat-report/vat-liabilities-payments/vat-liabilities-payments.component.ts

```diff
@@ -66,25 +64,23 @@ export class VatLiabilitiesPayments implements OnInit, OnDestroy {
     /** Holds Payment table columns */
     public paymentColumns: string[] = ["index", "received", "amount"];
     /** Holds Liability table columns */
-    public liabilityColumns: string[] = ["index", "from", "to", "originalAmount", "outstandingAmount", "due"];
+    public liabilityColumns: string[] = ["index", "from", "to", "originalAmount", "outstandingAmount", "type", "due"];
     /** Holds current table columns */
     public displayColumns: string[] = [];
     /** Observable to store true if API Call is in progress */
     public liabilityPaymentListInProgress$ = this.componentStore.select(state => state.liabilityPaymentListInProgress);
     /** Holds true if user in vat-payment */
     public isPaymentMode: boolean;
-    /** Company base currency symbol */
-    public baseCurrencySymbol: string = "";
+    /** Stores the current company */
+    public activeCompany: any = {};
     /** This will hold the value out/in to open/close setting sidebar popup */
     public asideGstSidebarMenuState: string = 'in';
 
     constructor(
         private activatedRoute: ActivatedRoute,
-        private gstReconcileService: GstReconcileService,
```

Comment:
Remove also in import for VatService, GstReconcileService

Reply:
Done